src/index.js

```diff
@@ -8,15 +8,22 @@ import { store } from '~/store'
 import loadSafesFromStorage from '~/routes/safe/store/actions/loadSafesFromStorage'
 import loadActiveTokens from '~/logic/tokens/store/actions/loadActiveTokens'
 import loadDefaultSafe from '~/routes/safe/store/actions/loadDefaultSafe'
+import loadCookiesFromStorage from '~/logic/cookies/store/actions/loadCookiesFromStorage'
 
 if (process.env.NODE_ENV !== 'production') {
   // eslint-disable-next-line
   const whyDidYouRender = require('@welldone-software/why-did-you-render')
   whyDidYouRender(React)
 }
 
+// $FlowFixMe
 store.dispatch(loadActiveTokens())
 store.dispatch(loadSafesFromStorage())
 store.dispatch(loadDefaultSafe())
+store.dispatch(loadCookiesFromStorage())
 
-ReactDOM.render(<Root />, document.getElementById('root'))
+const root = document.getElementById('root')
+
+if (root !== null) {
```

Comment:
why do you need this check?

Reply:
f we don’t do this, flow’s server will complain.
In this case root will never be null, but Flow doesn’t seem to know.